### ISSUE TITLE
Notice: Function Automattic\WooCommerce\Blocks\Integrations\IntegrationRegistry::register was called incorrectly. nets_easy is already registered.

### ISSUE THREAD

**Issue by kimhf** [krokedil/dibs-easy-for-woocommerce#258]
The changes made in commit 99f7adc are resulting in a lot of notices:

`Notice: Function Automattic\WooCommerce\Blocks\Integrations\IntegrationRegistry::register was called incorrectly. nets_easy is already registered.`

This code in `dibs-easy-for-woocommerce.php` seems to be the issue.

```
add_action(
	'woocommerce_blocks_payment_method_type_registration',
	function ( $payment_method_registry ) use ( $payment_methods ) {
		foreach ( $payment_methods as $payment_method ) {
			$payment_method_registry->register( new Nets_Easy_Checkout_Block( $payment_methods ) );
		}
	}
);
```

My guess is the block registration should not be done in a foreach loop and this loop is resulting in the same block being registered multiple times.

**Comment by MichaelBengtsson** [comment]
Yes you are correct. Seems like the foreach is no longer necessary with how we implemented it after adding support for the other gateways we have in the plugin either. So it can just be removed from my testing, i will do some more testing to ensure it works and then push a release out with the fix for this. Thank you for reporting this @kimhf !

**Comment by MichaelBengtsson** [comment]
@kimhf This has now been fixed and pushed in version 2.8.2 which is live now. Let me know if there are any other issues with this and i will take a look at it!
